README.md

```diff
@@ -90,4 +91,18 @@ textColor | String | Color of the value text | #000000
 strokeWidth | Number | Thickness of the arcs | 17
 valueDisplayFunction | Function | Custom function to alter the display text | `(v) => v`
 responsive | Boolean | Use `%` instead of `px` | false
-animation | Object | Animation config | {<br>animated: true,<br>animateValue: true,<br>animationDuration: 2000 (in ms),<br>animationFunction: 'ease-in-out'<br>}
+
+
+## Animation:
```

Comment:
Sorry, I should have been clearer in my last comment. Move this section up above the Properties table, and provide a HTML snippet of how to supply the animation settings.

Reply:
Ok i’ll Provide a snippet as soon as possible.